### PR TITLE
Updates to Jakarta Enterprise Web Services spec Chapter 9

### PR DESCRIPTION
**Pull request by sawadood**
The pull request is opened to Jakarta-ize chapter 9 of Jakarta Enterprise Web services specification.

### CODE REVIEW

File: enterprise-ws-spec/src/main/asciidoc/Security.adoc [hunk 1]
```diff
@@ -95,52 +94,52 @@ image:11.png[image,width=382,height=216]
...
-Server forwards authentication credentials into the Java EE Application
-Server. The Java EE application server handles the request similar to
+Server forwards authentication credentials into the Jakarta EE Application
+Server. The Jakarta EE application server handles the request similar to
 how it handles other HTTP requests.
 
 Two forms of authentication are available for use within Web Services
-for Java EE based on existing Java EE functionality. These are HTTP
+for Jakarta EE based on existing Jakarta EE functionality. These are HTTP
 BASIC-AUTH and Symmetric HTTP, which are defined by the Servlet
```
Comment (lukasj) [inline enterprise-ws-spec/src/main/asciidoc/Security.adoc:107]: Jakarta Servlet

File: enterprise-ws-spec/src/main/asciidoc/Security.adoc [hunk 2]
```diff
@@ -199,11 +198,11 @@ requiring the invention of special protocols.
...
-The server relies on the security infrastructure of the Java EE
+The server relies on the security infrastructure of the Jakarta EE
 Application Server.
 
 The Quality of Service (QoS) of a secure Web service container is based
-on the QoS requirements and functionality of the underlying Java EE
+on the QoS requirements and functionality of the underlying Jakarta EE
 application server itself (e.g., integrity).
 
 The server relies on HTTPS and RMI-IIOP over SSL for hop-by-hop
```
Comment (lukasj) [inline enterprise-ws-spec/src/main/asciidoc/Security.adoc:208]: RMI-IIOP over SSL should be removed as RMI as such is being removed in EE 9.

File: enterprise-ws-spec/src/main/asciidoc/Security.adoc (hunk 1 above)
Comment (sawadood) [reply enterprise-ws-spec/src/main/asciidoc/Security.adoc]: ok, will update as suggested.

**Comment by sawadood** [comment]
I have updated links so they work in both html and pdf documents and also made updates to address the two comments by lukasj.  Please review again and let me if there are any more changes needed.

**Pull request merged**